client/client.go

```diff
@@ -39,6 +46,16 @@ func Home(issuesRaw []database.Issue) func(c *gin.Context) {
 	}
 }
 
+// coverpage generates an HTML template for the cover image.
+// If the coverpage is valid, it returns an HTML string with an image tag.
+// If the coverpage is not valid, it returns an empty HTML string.
+func coverpage(coverpage sql.NullString) template.HTML {
+	if coverpage.Valid {
+		return template.HTML(fmt.Sprintf(`<img src=%v style="max-width: 40vw;">`, coverpage.String))
```

Comment:
You will get owned if the coverpage contains pretty much anything you wouldn't want

Reply:
Should it be sanitized both here and on the input when it is added to the database then?

Reply:
@foodelevator it has been done